.github/workflows/ci.yml

```diff
@@ -161,6 +172,10 @@ jobs:
     timeout-minutes: 60
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 100 # checkout more commits to be able to determine merge base for GIB
```

Comment:
Does this mean that PRs with more than 100 commits will fail to build?

Reply:
I changed this to 0. If we notice this is causing issues, we can address this later by doing it in two steps, first checking if we can find the base with N commits and if that fails fetching all of them.

Reply:
FYI generally doing a full clone is much faster than fetch unless you are constrained by bandwidth. (anecdotal experience as well as an old blog from GitHub).